jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -4,10 +4,11 @@
 
 (ros::load-ros-package "fetcheus")
 (ros::load-ros-package "fetch_driver_msgs")
+(ros::load-ros-package "robot_controllers_msgs")
 
 (defclass fetch-interface
   :super robot-move-base-interface
-  :slots (gripper-action moveit-robot)
+  :slots (gripper-action moveit-robot controller-action)
```

Comment:
`robot-interface` has `controller-actions` slot already, which is quite close to `controller-action`.
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L101

so how about changing the name to somthing like `fetch-controller-action` or `robot-controller-action` or something...

Reply:
Thank you for advice.
I rename `controller-action` -> `fetch-controller-action`.